docs/examples.rst

```diff
@@ -471,3 +471,38 @@ like so:
 .. include::  ../tests/core/contracts/test_contract_example.py
     :code: python
     :start-line: 1
+    
+Using Infura Rinkeby Node
+=========================
+Import your required libraries
+
+.. code-block:: python
+
+    from web3 import Web3, HTTPProvider
+    
+Intialize your infura node in a variable
```

Comment:
I would say: "Initialize a web3 instance with an Infura node"

Reply:
yours sounds better ;)

it should definitely be "web3 instance"